vru_use_case_scripts/calc_post_encroachment_time

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python3
+
+from argparse import ArgumentParser
+from pathlib import Path
+
+from matplotlib import pyplot as plt
+import numpy as np
+import pandas as pd
+from scipy.interpolate import splrep, BSpline
+from shapely import Point, LineString
+import shapely.geometry
+import shapely.plotting
+
+
+def parse_vehicle_odometry_from_csv(odometry_csv):
+    return pd.read_csv(odometry_csv)
+
+
+def parse_pedestrian_odometry_from_csv(odometry_csv):
+    return pd.read_csv(odometry_csv)
+
+
+def get_encroachment_zone(vehicle_odom_df, pedestrian_odom_df):
+    vehicle_path = LineString(
+        [
+            Point(row["Map Position X (m)"], row["Map Position Y (m)"])
+            for _, row in vehicle_odom_df[
+                ["Map Position X (m)", "Map Position Y (m)"]
+            ].iterrows()
+        ]
+    )
+
+    pedestrian_path = LineString(
+        [
+            Point(row["Map Position X (m)"], row["Map Position Y (m)"])
+            for _, row in pedestrian_odom_df[
+                ["Map Position X (m)", "Map Position Y (m)"]
+            ].iterrows()
+        ]
+    )
+
+    intersect_point = vehicle_path.intersection(pedestrian_path)
+
+    plt.plot(*vehicle_path.xy)
+    plt.plot(*pedestrian_path.xy)
+
+    BOX_SIZE = 2
```

Comment:
Comment that this is the variable for configuring "encroachment box" (just made up that term)